docs/networking.md

```diff
@@ -19,14 +27,11 @@ The following table provides the support status for various networking providers
 | Romana | 1.8 | - | 1.18 | 1.19 |
 | Weave | 1.5 | - | - | - |
 
+### Which networking provider should you use?
 
-The networking options determines how the pod and service networking is implemented and managed.
-
-Kubernetes Operations (kops) currently supports 3 networking modes:
+Kops maintainers have no bias over the CNI provider that you run, we only aim to be flexible and provide a working setup of the CNIs.
 
-* `kubenet` Kubernetes native networking via a CNI plugin.  This is the default.
-* `cni` Container Network Interface(CNI) style networking, often installed via a Daemonset.
-* `external` networking is done via a Daemonset. This is used in some custom implementations.
+We do recommended something other than `kubenet` for production clusters due to `kubenet`'s limitations, as explained [below][#kubenet-default].
```

Comment:
I do believe the distinction between kubenet-based providers and CNI-based providers is important to document.

Reply:
@johngmyers I felt I did? Kubenet is still in an entirely separate section from CNI and carries appropriate warnings.